src/Generation/Generator/Renderer/Public/FunctionRenderer.cs

```diff
@@ -79,8 +79,14 @@ private static string RenderCallStatement(GirModel.Function function, IReadOnlyL
 
         call.Append($"{Namespace.GetInternalName(function.Namespace)}.{parent}.{Function.GetName(function)}(");
         call.Append(string.Join(", ", parameters.Select(x => x.GetCallName())));
+        if (function.Throws)
+            call.Append(Error.RenderParameter(function));
+
         call.Append(");\n");
 
+        if (function.Throws)
```

Comment:
Remove `if` (add parameter `Callable` to the method and check inside wether to print or not).